webapp/src/Form/Type/ProblemUploadType.php

```diff
@@ -28,6 +29,10 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'accept' => 'application/zip',
             ],
         ]);
+        $builder->add('delete_data_first', CheckboxType::class, [
+            'help' => 'If checked, all existing testcases and attachments for the problem will be deleted, if any. Only useful for existing problems.',
```

Comment:
what about resetting other things like validators/validator args, and limits?

Reply:
Aren’t they always overwritten? If not this makes sense to do and I will do ir. Any more we want to do?

Reply:
I don't think so. You should give this a try. And perhaps add a test :-D

Reply:
🤣 we should really do the tests